packages/block-editor/src/components/block-compare/index.js

```diff
@@ -60,45 +65,36 @@ class BlockCompare extends Component {
 			rawContent: newContent.join( '' ),
 			renderedContent,
 		};
-	}
+	};
 
-	render() {
-		const {
-			block,
-			onKeep,
-			onConvert,
-			convertor,
-			convertButtonText,
-		} = this.props;
-		const original = this.getOriginalContent( block );
-		const converted = this.getConvertedContent( convertor( block ) );
-		const difference = this.getDifference(
-			original.rawContent,
-			converted.rawContent
-		);
+	const original = getOriginalContent();
```

Comment:
Why does this need to be a function?

Reply:
Yes I can remove this function it's no longer relevant.